### ISSUE TITLE
iron-selector accessibility issue

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-selector#21]
Keyboard navigation isn't working on this page. I am unable to navigate to the items and select them using tab / arrows / enter. I can do this with a screen reader, but not just with keyboard.

**Comment by morethanreal** [comment]
Will not fix this one, because `iron-selector` is an abstract element and not a UI control by itself.

**Comment by lpalmaro** [comment]
Got it - thanks for the clarification!
